data/os/RedHat.yaml

```diff
@@ -0,0 +1,2 @@
+---
+nftables::nft_path: /sbin/nft
```

Comment:
```suggestion
nftables::nft_path: /usr/sbin/nft
```
since
```
rpm -ql nftables | grep bin
/usr/sbin/nft
```

Reply:
ugh damn :D 
I copied that path from the old code. I guess it only works bebause of the /sbin -> /usr/sbin symlink.